_includes/scenarios/integrate/data-model-overview.md

```diff
@@ -241,139 +241,257 @@ With the information in this type you can clearly identify the Transport object
+| query | [`IdentityAttributeQuery`](#identityattributequery) \| [`ThirdPartyRelationshipAttributeQuery`](#thirdpartyrelationshipattributequery) | The query the Attribute that is listened to must match. Note that you cannot send a [`RelationshipAttributeQuery`](#relationshipattributequery) here, because it doesn't make sense: by definition, both parties know about a RelationshipAttribute right from the beginning, because one party requests its creation, and the other one accepts it. |
+| peer  | `string`                                                                                                                               | The Address of the peer that requested the Attribute Listener.                                                                                                                                                                                                                                                                                       |
 
 # Content Types
 
 Content Types can be seen as a data contract between Identities. The medium through which this data is exchanged are the [Transport types](#transport-types) (e.g. Messages, Tokens, ...). This chapter shows all the Content types and describes their intended usage.
 
 ## Request
 
-A Request allows you to ask another Identity to do something. What this "something" is depends on which of the so called [Request Items](#requestitem) were added to the Request (e.g. [`CreateAttributeRequestItem`]({% link _docs_integrate/requests-and-requestitems.md %}#createattributerequestitem), [`ReadAttributeRequestItem`]({% link _docs_integrate/requests-and-requestitems.md %}#readattributerequestitem), ...). The Request is then sent to the peer via Message or Relationship Template. The peer can then review the Request and decide whether they want to accept or reject it. And if they accept it, they can even choose which of the Items they want to accept. You can also put multiple Items into a [group](#requestitemgroup) in order to ensure that they can only be accepted/rejected as a unit.
+A Request allows you to ask another Identity to do something. What this "something" is depends on which of the so called [RequestItems](#requestitem) were added to the Request (e.g. [`CreateAttributeRequestItem`]({% link _docs_integrate/requests-and-requestitems.md %}#createattributerequestitem), [`ReadAttributeRequestItem`]({% link _docs_integrate/requests-and-requestitems.md %}#readattributerequestitem), ...). The Request is then sent to the peer via Message or Relationship Template. The peer can then review the Request and decide whether they want to accept or reject it. And if they accept it, they can even choose which of the Items they want to accept. You can also put multiple Items into a [group](#requestitemgroup) in order to ensure that they can only be accepted/rejected as a unit.
```

Comment:
> `[RequestItems](#requestitem)`

isn't this a dead link now? I think it should be `[RequestItems](#requestitems)`. Can you double-check these new titles everywhere?